CONTRIBUTING.md

```
@@ -0,0 +1,129 @@
+it("respects --json on overview inputs subcommand", async () => {
+  // ...
+});
+```
+
+## Pull requests
+
+### Commit and PR titles
+
+PR titles follow [Conventional Commits](https://www.conventionalcommits.org/), with an optional scope:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)